examples/mixhop/mixhop_trainer.py

```diff
@@ -0,0 +1,135 @@
+import os
+
+os.environ['CUDA_VISIBLE_DEVICES'] = '0'
+os.environ['TL_BACKEND'] = 'tensorflow'
+# os.environ['TL_BACKEND'] = 'torch'
+# os.environ['TL_BACKEND'] = 'paddle'
```

Comment:
Comment out this part of the code and leave it up to the user to choose. And remove redundant code.

Reply:
ok